doc/connectivity/bluetooth/api/audio/bluetooth-le-audio-arch.rst

```diff
@@ -59,6 +59,292 @@ GAF has been implemented in Zephyr with the following structure.
+   | GATT sub-procedure               | AICS | ASCS | BASS | CAS | CSIS | HAS | MCS | MICS | PACS | TBS | VCS | VOCS | TMAS |
+   +==================================+======+======+======+=====+======+=====+=====+======+======+=====+=====+======+======+
+   | Write Characteristic Value       | M    | M    | M    |     | C.2  | C.3 | M   | M    | M    | M   | M   | M    |      |
+   +----------------------------------+------+------+------+-----+------+-----+-----+------+------+-----+-----+------+------+
+   | Write Without Response           |      | M    | M    |     |      |     | M   |      |      | M   |     |      |      |
+   +----------------------------------+------+------+------+-----+------+-----+-----+------+------+-----+-----+------+------+
+   | Notifications                    | M    | M    | M    |     | C.2  | C.4 | M   | M    | M    | M   | M   | M    |      |
+   +----------------------------------+------+------+------+-----+------+-----+-----+------+------+-----+-----+------+------+
+   | Indications                      |      |      |      |     |      | C.3 |     |      |      |     |     |      |      |
+   +----------------------------------+------+------+------+-----+------+-----+-----+------+------+-----+-----+------+------+
+   | Read Characteristic Descriptors  | M    | M    | M    |     | C.2  | C.3 | M   | M    | M    | M   | M   | M    |      |
+   +----------------------------------+------+------+------+-----+------+-----+-----+------+------+-----+-----+------+------+
+   | Write Characteristic Descriptors | M    | M    | M    |     | C.2  | C.3 | M   | M    | M    | M   | M   | M    |      |
+   +----------------------------------+------+------+------+-----+------+-----+-----+------+------+-----+-----+------+------+
+   | Write Long Characteristic Value  |      | M    | C.1  |     |      |     |     |      |      |     |     |      |      |
+   +----------------------------------+------+------+------+-----+------+-----+-----+------+------+-----+-----+------+------+
+   | Read Long Characteristic Value   |      | C.5  | C.5  |     |      |     | C.5 |      |      | C.5 |     |      |      |
+   +----------------------------------+------+------+------+-----+------+-----+-----+------+------+-----+-----+------+------+
+
+Notes:
+
+* M Mandatory
+* C.1 Mandatory if the Add Source operation is supported
+* C.2 Mandatory if Set Membert Lock characteristic is supported
+* C.3 Mandatory if the Hearing Aid Present Control Point characteristic is supported
+* C.4 Mandatory if the Hearing Aid Present Control Point characteristic and EATT are both supported
+* C.5 Mandatory if it supports characteristic values larger than the ATT_MTU
+
+Profile Dependencies
+====================
+
+Besides depending on GATT several of the LE Audio profiles depend on other profiles and services, as outlined in the following tables.
```

Comment:
Perhaps we should elaborate that any "Server" here refers to acting in the GATT server role while "Client" refers to a GATT client role for the specified services